client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -273,8 +282,13 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
 
     protected createViewModelWithTitles(model: Motion, initialLoading: boolean): ViewMotion {
         const viewModel = super.createViewModelWithTitles(model, initialLoading);
+
         viewModel.getIdentifierOrTitle = () => this.getIdentifierOrTitle(viewModel);
         viewModel.getProjectorTitle = () => this.getAgendaSlideTitle(viewModel);
+        if (viewModel.isParagraphBasedAmendment()) {
+            viewModel.diffLines = this.getAmendmentParagraphs(viewModel, this.motionLineLength, false);
```

Comment:
Please do the following:
- add `public getDiffLines: () => DiffLinesInParagraph[];` to view-motion.ts
- change this line to `viewModel.diffLines = () => getAmendmentParagraphs(viewModel, this.motionLineLength, false);`
- Create a getter in view-motion.ts: `public get diffLines():  DiffLinesInParagraph[] { return this.getDiffLines(); }`

This should fix the issue you described and it should not be necessary to change the method `getAmendmentParagraphs`.

Reply:
Also do dome "error handling" if these functions/getter in `ViewMotion` are called, if the motion is not a paragraph based amendment.